cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/SymbolResolver.kt

```diff
@@ -492,6 +492,15 @@ open class SymbolResolver(ctx: TranslationContext) : ComponentPass(ctx) {
 
         // We also set the callee's refersTo
         callee.refersTo = call.invokes.firstOrNull()
+        val refersToName = callee.refersTo?.name
```

Comment:
You need to revert the symbol resolver back to main. It looks like you merged in an old version of the #2085 branch